### PR TITLE
Fix typos in 2.0-preview README

### PR DESCRIPTION
**Pull request by cletusw**

### CODE REVIEW

**Comment by cletusw** [comment]
@kevinpschaaf Please review, thanks!

File: README.md [hunk 1]
```diff
@@ -256,12 +256,12 @@ id is to use `id`.
...
 * `lazyRegister` option removed and is now “on” by default
 * Experimental: `listeners` and `hostAttributes` are deferred until "afterNextRender", since the majority uses of these should not be initial paint-blocking.  Please help identify use cases where paint-blocking host attributes/listeners are useful/needed.
-* Requst to early-users: We would really like to remove the `ready` callback, since its use is generally anti-pattern-ish, and it's hard to document when a "one-shot callback that runs after all local dom & observers have flushed" should actually be used, as opposed to running said code in an observer.  In exploring alacarte, please try to avoid `ready` and help identify use cases where it is useful/needed.
+* Request to early-users: We would really like to remove the `ready` callback, since its use is generally anti-pattern-ish, and it's hard to document when a "one-shot callback that runs after all local dom & observers have flushed" should actually be used, as opposed to running said code in an observer.  In exploring a la carte, please try to avoid `ready` and help identify use cases where it is useful/needed.
```
Comment (arthurevans) [inline README.md]: alacarte here should be "Polymer 2.0"

**Comment by arthurevans** [comment]
One nit otherwise LGTM. Somehow I don't think that this is breaking the CI build...

**Comment by cletusw** [comment]
@arthurevans Made requested change.

**Comment by kevinpschaaf** [comment]
LGTM, thank you!

**Pull request merged**